src/py42/sdk/queries/fileevents/filters/exposure_filter.py

```diff
@@ -129,7 +129,7 @@ class TabURL(FileEventFilterStringField):
     contents were read by the browser (applies to ``read by browser or other app`` events only).
```

Comment:
Can we adjust the doc str to be something like

```
Class that filters events based on all the URLs of the browser tabs at the time the file
    contents were read by the browser (applies to ``read by browser or other app`` events only).
```

Reply:
Changed.